docs/source/examples/Widget Styling.ipynb

```diff
@@ -1046,19 +1046,6 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## `Image` layout and sizing\n",
-    "\n",
-    "The layout and sizing of images is a little different than for other elements for a combination of historical reasons (the HTML tag `img` existed before CSS) and practical reasons (an image has an intrinsic size).\n",
-    "\n",
-    "Sizing of images is particularly confusing because there are two plausible ways to specify the size of an image. The `Image` widget has attributes `width` and `height` that correspond to attributes of the same name on the HTML `img` tag. In addition, the `Image` widget, like every other widget, has a `layout`, which also has a `width` and `height`.\n",
-    "\n",
-    "In addition, some CSS styling is applied to images that is not applied to other widgets: `max_width` is set to `100%` and `height` is set to `auto`. \n",
-    "\n",
-    "You should not rely on `Image.width` or `Image.height` to determine the display width and height of the image widget. Any CSS styling, whether via `Image.layout` or some other source, will override `Image.width` and `Image.height`.\n",
-    "\n",
-    "When displaying an `Image` widget by itself, setting `Image.layout.width` to the desired width will display the `Image` widget with the same aspect ratio as the original image.\n",
-    "\n",
-    "When placing an `Image` inside a `Box` (or `HBox` or `VBox`) the result depends on whether a width has been set on the `Box`. If it has, then the image will be stretched (or compressed) to fit within the box because `Image.layout.max_width` is `100%` so the image fills the container. This will usually not preserve the aspect ratio of image.\n",
     "\n",
```

Comment:
Can you delete this newline as well, so the cell starts with the next header?

Reply:
Yep just deleted the newline at the end and pushed the commit.
Let me know if there is anything else / if I did something incorrectly!